charts/hono/templates/hono-service-device-registry-base/hono-service-device-registry-amqps-route.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if and .Values.deviceRegistryExample.enabled ( eq .Values.platform "openshift" ) }}
 #
-# Copyright (c) 2019 Contributors to the Eclipse Foundation
+# Copyright (c) 2019, 2020 Contributors to the Eclipse Foundation
```

Comment:
if you do not change anything in the file then you do not need to update the copyright header FMPOV

Reply:
I added some changes and after refactoring I have removed it but didn't update he copyright header. I will revert it.

Reply:
Note that this affects multiple files ...

Reply:
Thanks for pointing out and I have updated those files and pushed a commit.